internal/broker/bind_get_test.go

```diff
@@ -45,4 +52,39 @@ func TestGetBinding(t *testing.T) {
 		errorResponse := apiErr.ErrorResponse().(apiresponses.ErrorResponse)
 		require.Equal(t, "Binding expired", errorResponse.Description)
 	})
+
+	t.Run("should return 404 code for the binding of deprovisioned/suspended instance", func(t *testing.T) {
```

Comment:
Could we extend the case by first showing that binding was retrieved from GET and then changing type to deprovisioned resulted in 404?

Reply:
I deliberately decided not to do this.

Reply:
Discussed in direct communication.